google/cloud/storage/object_stream.h

```diff
@@ -148,10 +152,19 @@ class ObjectReadStream : public std::basic_istream<char> {
    */
   std::string const& computed_hash() const { return buf_->computed_hash(); }
 
-  /// The headers returned by the service, for debugging only.
-  std::multimap<std::string, std::string> const& headers() const {
-    return buf_->headers();
-  }
+  /**
+   * The headers (in any) returned by the service, for debugging only.
```

Comment:
s/in any/if any/
s/, for/. For/

Reply:
Fixed.

Reply:
Nit: Looks like you missed the `s/in any/if any/` part.

Reply:
Doh!